### PR TITLE
Migrate from tslint to eslint

### PR DESCRIPTION
**Pull request by KaeruCT**
Solves #1 

Do let me know if this can be improved further :)

### CODE REVIEW

**Review by PierreBresson — APPROVED** [review]

**Review by PierreBresson — CHANGES_REQUESTED** [review]
Thanks for your help! Looks like the tests are failing but otherwise looks good

File: package.json [hunk 1]
```diff
@@ -40,13 +40,17 @@
   "devDependencies": {
     "@types/jest": "25.2.1",
     "@types/ramda": "0.27.4",
+    "@typescript-eslint/eslint-plugin": "4.3.0",
+    "@typescript-eslint/parser": "4.3.0",
+    "eslint": "7.10.0",
+    "eslint-config-prettier": "6.12.0",
+    "eslint-plugin-jest": "^24.0.2",
```
Comment (PierreBresson) [inline package.json]: Could you remove all ^ ? So we all have the same versions

**Comment by KaeruCT** [comment]
Thanks for the quick response! I have done the changes requested and tests should be passing now.

Since I found this issue through the  `hacktoberfest` label, would you mind adding the `hacktoberfest` topic to this repo? It turns out you now need the `hacktoberfest` topic on your repo to be eligible. Thanks in advance :)

**Comment by PierreBresson** [comment]
> Thanks for the quick response! I have done the changes requested and tests should be passing now.
> 
> Since I found this issue through the `hacktoberfest` label, would you mind adding the `hacktoberfest` topic to this repo? It turns out you now need the `hacktoberfest` topic on your repo to be eligible. Thanks in advance :)

Done ;)

**Review by PierreBresson — APPROVED** [review]
Merci !

**Pull request merged**